### PR TITLE
[Merged by Bors] - fix: use crate version for user agent

### PR DESCRIPTION
**Pull request by EstebanBorai**
Uses the version from Cargo.toml to determine the default `User-Agent` version

### CODE REVIEW

**Comment by EstebanBorai** [comment]
@ajhunyady I will get some tests on this before asking for review!

File: tests/integration-sends-data-via-post.bats [hunk 1]
```diff
@@ -49,3 +49,19 @@ teardown() {
...
+    echo "Produce \"North Carolina\" on $TOPIC"
+    echo "North Carolina" | fluvio produce $TOPIC
+
+    echo "Sleep to ensure record is processed"
+    sleep 25
+
+    echo "Contains User Agent with current version"
+    cat ./$LOGGER_FILENAME | grep "user_agent: \"fluvio/http-sink $VERSION\""
+    assert_success
+}
```
Comment (EstebanBorai) [inline tests/integration-sends-data-via-post.bats:67]: New test to ensure the `User-Agent` uses the same version as connector

**Review by morenol — APPROVED** [review]

**Pull request closed without merging**